creator-node/src/utils/legacyUtils.js

```diff
@@ -222,14 +225,14 @@ export async function findCIDInNetwork(
           )
         },
         logger,
-        logLabel: 'fetchFileFromNetworkAndWriteToDisk',
+        logLabel: 'findCIDInNetwork',
         options: {
           retries: numRetries,
           minTimeout: 3000
         }
       })
 
-      return true
+      found = true
```

Comment:
i think this is actually incorrect since now even if it is found, it'll continue through for loop for other nodes - was previously correct to return as soon as one succeeds

i think the correct change here would just be to not set `found = await asyncRetry()` since that does nothing

wdyt?

Reply:
oh right 🤦‍♀️ thats what I wrote initially. 👍